jobs/integr8ly/ocp3/test-suites/all-tests-executor.yaml

```diff
@@ -103,12 +103,8 @@
         try {
             timeout(180) { ansiColor('gnome-terminal') { timestamps {
                 node('cirhos_rhel7') {
-                    String description = "cluster url: ${CLUSTER_URL}<br>testing branch: ${BRANCH}<br>" + 
-                        "evals username: ${EVALS_USERNAME}"
-                    if (currentBuild.getBuildCauses('hudson.model.Cause$UserIdCause')['userId']){
-                        description += "<br>triggered by: " + currentBuild.getBuildCauses('hudson.model.Cause$UserIdCause')['userId']
-                    }
-                    currentBuild.description = description
+                    currentBuild.displayName = "${currentBuild.displayName} ${CLUSTER_URL}"
+                    currentBuild.description = "testing branch: ${BRANCH}<br>evals username: ${EVALS_USERNAME}"
```

Comment:
So here the name is `BRANCH` ? :) It's not `INSTALLATION_BRANCH` right :)

Reply:
https://github.com/integr8ly/ci-cd/pull/286/files#diff-ab9ffde24a59166ffe8a781a189c070dR83

Reply:
yes. here the param is `BRANCH`

Reply:
Yes, it is indeed `BRANCH` here.